dashboard/app/views/levels/_lab2.html.haml

```diff
@@ -0,0 +1,8 @@
+#codeApp{style: "position: absolute; top: 60px; bottom: 10px; left: 10px; right: 10px"}
+  #lab2-container{style: "height: 100%"}
+
+- content_for(:head) do
+  %script{src: webpack_asset_path("js/#{js_locale}/music_locale.js")}
+  %script{src: webpack_asset_path("js/#{js_locale}/standaloneVideo_locale.js")}
+  %script{src: webpack_asset_path('js/googleblockly.js')}
+  = render_app_dependencies
```

Comment:
Do we still need `render_app_dependencies` here since we're already loading blockly and locale?

Reply:
Good question; it was possible to remove this dependency.